lib/termineter/modules/brute_force_login.py

```diff
@@ -57,61 +58,51 @@ def __iter__(self):
 				yield password
 				password = self.dictionary.readline()
 			self.dictionary.close()
-		raise StopIteration
+		#raise StopIteration; seems to have broken brute forcing multiple user id's
 
 def from_hex(data):
 	return binascii.a2b_hex(data)
 
 def to_hex(data):
 	return binascii.b2a_hex(data).decode('utf-8')
 
+
+
 class Module(TermineterModuleOptical):
 	connection_state = TermineterModuleOptical.connection_states.none
 	def __init__(self, *args, **kwargs):
 		TermineterModuleOptical.__init__(self, *args, **kwargs)
-		self.author = ['Spencer McIntyre']
+		self.author = ['Spencer McIntyre, David Garlak (@vexance)']
```

Comment:
This should be an array of strings.

```suggestion
		self.author = ['Spencer McIntyre', 'David Garlak (@vexance)']
```

Reply:
Fixed in the latest commit